src/test_typing_extensions.py

```diff
@@ -103,6 +103,11 @@
     runtime_checkable,
 )
 
+if sys.version_info >= (3, 14):
+    from test.support import EqualToForwardRef
```

Comment:
this is going to mean that the tests can't be run on e.g. a Conda build -- Conda builds don't include the `test` module in the stdlib

Reply:
I guess I can inline it